pom.xml

```diff
@@ -1333,6 +1346,7 @@
                 <skipTests>true</skipTests>
                 <skipITs>false</skipITs>
                 <skipClientITs>false</skipClientITs>
+                <argLine>-Xmx2048m</argLine>
```

Comment:
Is this necessary if the new test is being run in non-confidential-tests instead?

Reply:
Yes